live/src/main/scala/org/dbpedia/extraction/mappings/AbstractExtractorWikipedia.scala

```diff
@@ -0,0 +1,21 @@
+package org.dbpedia.extraction.mappings
+
+import org.dbpedia.extraction.ontology.Ontology
+import org.dbpedia.extraction.util.Language
+import scala.language.reflectiveCalls
+
+/**
+  * Overwrites the apiUrl property for the AbstractExtractor in the core module to work with configurable Wikipedias.
+  * @author Lukas Faber, Stephan Haarmann, Sebastian Serth
+  * date 21.05.2016.
+  */
+class AbstractExtractorWikipedia(
+  context : {
+    def ontology : Ontology
+    def language : Language
+  })
+  extends AbstractExtractor (context)
+{
+
+  override def apiUrl: String = "https://" + context.language.wikiCode + ".wikipedia.org/w/api.php"
```

Comment:
we have the exact same class in the server module
https://github.com/dbpedia/extraction-framework/blob/master/server/src/main/scala/org/dbpedia/extraction/mappings/AbstractExtractorWikipedia.scala

maybe it makes sense to move that to core and delete this one, wdyt?